ppl/zqd/pcapanalyzer/process.go

```diff
@@ -15,9 +15,9 @@ var ErrNotFound = errors.New("executable not found")
 
 // Process is an interface for interacting running with a running process.
 type ProcessWaiter interface {
-	// Wait waits for a running process to exit, returning any errors that
-	// occur.
-	Wait() error
+	// Wait waits for a running process to exit, returning the
+	// process' accumulated stdout and any errors that occur.
+	Wait() (string, error)
```

Comment:
I'm somewhat nervous about grabbing stdout automatically. How about if we only store it if desired (like a flag to NewProcess), and add another method on ProcessWaiter that returns the captured stdout, or `""`?

Reply:
Sounds good - are you nervous for memory reasons?

Reply:
Yes; I don't want to accidentally read a large output to memory.

Reply:
88775ab